server/game/core/Player.js

```diff
@@ -159,7 +182,7 @@ class Player extends GameObject {
      * @param { WildcardZoneName.AnyArena | ZoneName.GroundArena | ZoneName.SpaceArena } arena Arena to select units from
      */
     getOtherUnitsInPlayWithTrait(ignoreUnit, trait, arena = WildcardZoneName.AnyArena) {
-        return this.getArenaCards(arena).filter((card) => card.isUnit() && card !== ignoreUnit && card.hasSomeTrait(trait));
+        return this.getArenaCards({ otherThan: ignoreUnit, trait, arena }).filter((card) => card.isUnit() && card !== ignoreUnit && card.hasSomeTrait(trait));
```

Comment:
Do we have any sort of intellisense for these filter properties? I saw them in the other file where they were declared, and this is how i wish the arena/zone filter/searching had worked, I just want to make sure it's obvious what we have available to us.